views/audit/big_query_elt_script_logging.sql

```diff
@@ -0,0 +1,785 @@
+WITH jobChangeEvent AS (
```

Comment:
wrap the whole SQL in CREATE OR REPLACE VIEW and add README steps to replace view name with user-desired value